R/data.table.R

```diff
@@ -1646,14 +1646,14 @@ replace_dot_alias = function(e) {
           if (dotN(q)) return(TRUE) # For #5760
           # run GForce for simple f(x) calls and f(x, na.rm = TRUE)-like calls where x is a column of .SD
           # is.symbol() is for #1369, #1974 and #2949
-          if (!(is.call(q) && is.symbol(q[[1L]]) && is.symbol(q[[2L]]) && (q1c <- as.character(q[[1L]])) %chin% gfuns)) return(FALSE)
```

Comment:
@mattdowle I believe here, note the comment about is.symbol

Reply:
@MichaelChirico Thanks! This is a convoluted one as it's inside !() so it's really `(!is.call(q) || !is.symbol(q[[1L]])) && ...`.  Haven't looked as the issues yet but it seems different use case than sub_is_functioon.  Btw, what about a name like `is_call_to` instead of `sub_is_fun`. The one function `is_call_to(e, f)` could accept length f 1 or >1 so we don't have to use `sub_in_funs` instead.